src/porepy/models/abstract_equations.py

```diff
@@ -183,3 +184,57 @@ def perturbation_from_reference(self, variable_name: str, grids: list[pp.Grid]):
         d_var = var(grids) - var_ref(grids)
         d_var.set_name(variable_name + "_perturbation")
         return d_var
+
+
+class BoundaryConditionsMixin:
+    """TODO"""
+
+    mdg: pp.MixedDimensionalGrid
+
+    def update_boundary_conditions(self, initial: bool) -> None:
```

Comment:
I think "set_boundary_condition" is even more precise. This is already reflected in the docstring.

Should this be an abstract method?

Reply:
I agree